doc/README.hi.md

```diff
@@ -0,0 +1,143 @@
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_10.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_10.png)
+[<img src="../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png" width=405>](../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png)
+[<img src="../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png" width=405>](../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png)
+
+## विवरण
+
+NewPipe ना ही किसी Google फ्रेमवर्क लाइब्रेरी, और ना ही YouTube के API का इस्तेमाल करता है। इस ऐप को चलाने के लिए सिर्फ वेबसाइट्स से जानकारी प्राप्त करने की ज़रूरत है, तो इस ऐप का इस्तेमाल उन डिवाइसों पर भी किया जा सकता है जिनपर Google की सेवाएँ स्थापित नहीं हैं। और NewPipe जैसे कॉपीलेफ्ट किए गए मुक्त सॉफ़्टवेयर का इस्तेमाल करने के लिए आपको YouTube खाते की ज़रूरत नहीं।
+
+### सुविधाएँ
+
+* वीडियो खोजें
+* खाते की ज़रूरत नहीं
+* वीडियो के बारे में साधारण जानकारी पाएँ
+* YouTube के वीडियो देखें
+* YouTube के वीडियो सुनें
+* वीडियो देखते हुए दूसरे ऐप्स का इस्तेमाल करें (फ़्लोटिंग प्लेयर)
+* वीडियो देखने के लिए स्ट्रीम करने का प्लेयर चुनें
+* वीडियो डाउनलोड करें
+* सिर्फ ऑडियो डाउनलोड करें
+* Kodi में वीडियो को खोलें
+* अगले/संबंधित वीडियो देखें
+* YouTube को किसी विशिष्ट भाषा में खोजें
+* उम्र से जुड़ी पाबंदी वाले वीडियी देखें/छिपाएँ
+* चैनलों के बारे में साधारण जानकारी पाएँ
+* चैनल खोजें
+* किसी विशिष्ट चैनल से वीडियो देखें
+* Orbot/Tor के साथ चलाएँ (मुश्किलें आ सकती हैं)
+* 1080p/2K/4K में देखें
+* ऐप आपके देखे गए वीडियो याद रखेगी
+* चैनलों पर सदस्यता लें
+* देखे गए वीडियो खोजें
+* प्लेलिस्ट्स खोजें/देखें
+* प्लेलिस्ट्स को सूची में जोड़कर देखें
+* वीडियों को सूची में डालें
+* खुदकी प्लेलिस्ट्स बनाएँ
+* उपशीर्षक
+* लाइवस्ट्रीम करें/ देखें
+* वीडियो पे टिप्पणियाँ देखें
+
+### उपलब्ध सेवाएँ
+
+NewPipe पर कई सेवाएँ उपलब्ध हैं। हमारे [प्रलेख](https://teamnewpipe.github.io/documentation/) में यह बताया गया है कि ऐप और एक्सट्रैक्टर में एक नई सेवा कैसे जोड़ी जा सकती है। अगर आप कोई नई सेवा जोड़ना चाहते हैं, कृपया हमसे संपर्क करें। वर्तमान में उपलब्ध सेवाएँ हैं:
+
+* YouTube
+* SoundCloud \[बीटा\]
+* media.ccc.de \[बीटा\]
+* PeerTube instances \[बीटा\]
+* Bandcamp \[बीटा\]
+
+<!-- Hidden span to keep old links compatible. -->
+<span id="updates"></span>
+
+## स्थापित करना और अपडेट करना
+आप इनमें से किसी एक तरीके से NewPipe को स्थापित कर सकते हैं:
+ 1. हमारे अपने रिपॉज़िटरी को F-Droid पर जोड़ें और उसे वहाँ से स्थापित करें। अनुदेश यहाँ हैं: https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/
+ 2. [GitHub प्रकाशन](https://github.com/TeamNewPipe/NewPipe/releases) से APK डाउनलोड करें और उसे स्थापित करें।
+ 3. F-Droid के ज़रिए अपडेट करें। यह अपडेट करने का सबसे धीमा तरीका है, क्योंकि पहले F-Droid को बदलाव पहचानने होंगे, फिर वह APK बनाएगा, उसपर हस्ताक्षर करेगा, और आखिर में उपयोगकर्ताओं को अपडेट पुश करेगा।

```

Comment:
That's a fantastically accurate translation of "user". Nice one!

BTW, is "upyogiyon" incorrect? If not, I think that would be a bit easier to understand.

Reply:
Well no, "उपयोगी" means 'useful' rather than 'user'.